tests/apichecks/testdata/exceptions/acronyms.txt

```diff
@@ -114,6 +114,7 @@
 [acronyms] crd=computefirewallpolicies.compute.cnrm.cloud.google.com version=v1beta1: field ".status.selfLinkWithId" should be ".status.selfLinkWithID"
 [acronyms] crd=computeforwardingrules.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.sourceIpRanges" should be ".spec.sourceIPRanges"
 [acronyms] crd=computeforwardingrules.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.sourceIpRanges[]" should be ".spec.sourceIPRanges[]"
+[acronyms] crd=computeforwardingrules.compute.cnrm.cloud.google.com version=v1beta1: field ".spec.target.googleApiBundle" should be ".spec.target.googleAPIBundle"
```

Comment:
I was trying to suggest renaming this field to APIBundle. Do you mind explaining why making this an exception? I think the googleAPIBundle is new, right?